lib/entry-browser-cjs.ts

```diff
@@ -2,9 +2,16 @@ import AlgoliaAnalytics from "./insights";
 import { getFunctionalInterface } from "./_getFunctionalInterface";
 import { getRequesterForBrowser } from "./utils/getRequesterForBrowser";
 import { getRequesterForNode } from "./utils/getRequesterForNode";
+import { processQueue } from "./_processQueue";
 import { RequestFnType } from "./utils/request";
 
-export { getRequesterForBrowser, getRequesterForNode };
+export {
+  getRequesterForBrowser,
+  getRequesterForNode,
```

Comment:
I don't understand why browser exports the node requester, but that seems to already have been the case?

Reply:
Clear it doesn't need to export it. It was there by mistake. I'll get rid of it in the v2.

Reply:
Is it still tree shaked off by builder?